src/main/resources/openapi/PrintProviderAPI.yaml

```diff
@@ -2,120 +2,149 @@ openapi: 3.0.0
 info:
   title: Print ProviderAPI
   version: '1.0'
-  description: API for Print providers to retrieve print requests and to send updates on print request statuses.
+  description: API Specification for sharing Models between EIP and Print suppliers that will be exchanged over SFTP.
   contact:
     name: Krister Bone
     email: krister.bone@valtech.com
 servers:
   - url: 'https://api.printprovider.int.erop.ierds.uk'
-paths:
-  '/printrequests/standard?maxResults={maxResults}':
-    parameters:
-      - schema:
-          type: string
-        name: maxResults
-        in: path
-        required: true
-        description: 'Max results to return between 1 and 1000. Default: 100'
-    get:
-      summary: Printer Service Provider Print Requests API
-      description: Returns print requests queued for Print Provider fullfilment.
-      tags: []
-      responses:
-        '200':
-          $ref: '#/components/responses/PrintRequestsResponse'
-        '':
-          $ref: '#/components/responses/PrintRequestsResponse'
-      operationId: get-print-requests
-  '/printrequests/{requestid}':
-    post:
-      summary: ''
-      operationId: post-print-request-requestid
-      responses:
-        '201':
-          description: Accepted
-      description: Updates the status of the print request.
-      requestBody:
-        $ref: '#/components/requestBodies/PrintRequestUpdate'
-    parameters:
-      - schema:
-          type: string
-        name: requestid
-        in: path
-        description: The print request identifier
-        required: true
-  /printrequests:
-    post:
-      summary: ''
-      operationId: post-print-requests
-      responses:
-        '201':
-          description: Accepted
-      description: Updates the statuses of the print requests.
-      requestBody:
-        $ref: '#/components/requestBodies/PrintRequestUpdate'
-    parameters: []
 components:
```

Comment:
I don't understand what these models are (in context of an SFTP interface). Who is using them? What for? And where has the design come from?

Reply:
I can add to the sequence diagram.